CMakeLists.txt

```
@@ -389,6 +393,83 @@ endif()
 
 # ##############################################################################
 
+# If cuDSS is enabled and environment variables are set, install the cuDSS version
+if(USE_CUDSS AND DEFINED ENV{CUDA_PATH} AND DEFINED ENV{CUDSS_PATH})
```

There is no reason to manually check the the `CUDA_PATH` or `CUDSS_PATH` env variables. CMake, cuda and cudss provide way of finding them, like:

```suggestion
if(USE_CUDSS)
  find_package(CUDAToolkit REQUIRED)
  find_package(cudss REQUIRED)
```